src/main/resources/public/css/stylesheet.css

```diff
@@ -16,7 +16,15 @@ input, textarea { display: table-cell; }
 
 textarea { 
     margin: 0; 
-    padding: 0; 
-    display: block; 
+    padding: 0;  
     font-size: 100%; 
-  } 
\ No newline at end of file
+  }
+
+#book_table {
+    text-align: center !important;
```

Comment:
I would try to avoid using !important if possible

Reply:
good catch, this has now been removed.